core/sail/spin/src/test/java/org/eclipse/rdf4j/sail/spin/SpifSailTest.java

```diff
@@ -73,6 +77,7 @@ public void tearDown()
 	}
 
 	@Test
+	@Ignore
```

Comment:
This method that uses a manifest should have been kept and extended rather than replaced.